pkg/cmd/roachtest/cluster.go

```diff
@@ -2121,6 +2121,10 @@ func (c *cluster) StartE(ctx context.Context, opts ...option) error {
 	args := []string{
 		roachprod,
 		"start",
+		// Enable Jemalloc profiles for all roachtests. This has only
+		// minimal performance imapct and significantly improves the
+		// troubleshootability of memory-related issues.
+		"-e", "MALLOC_CONF=prof:true",
```

Comment:
[nit] Slight aesthetic preference for using a helper here and using `args = append(args, helperFn)` below.

Reply:
can you show me what you mean?

Reply:
```diff
diff --git i/pkg/cmd/roachtest/cluster.go w/pkg/cmd/roachtest/cluster.go
index 7d3bb3a8e4..025bb24d61 100644
--- i/pkg/cmd/roachtest/cluster.go
+++ w/pkg/cmd/roachtest/cluster.go
@@ -2124,12 +2124,17 @@ func (c *cluster) StartE(ctx context.Context, opts ...option) error {
        }
        args = append(args, roachprodArgs(opts)...)
        args = append(args, c.makeNodes(opts...))
+       args = append(args, enableJemallocArgs()...)
        if !argExists(args, "--encrypt") && c.encryptDefault {
                args = append(args, "--encrypt")
        }
        return execCmd(ctx, c.l, args...)
 }

+func enableJemallocArgs() []string {
+       return []string{"-e", "MALLOC_CONF=prof:true"}
+}
+
 // Start is like StartE() except it takes a test and, on error, calls t.Fatal().
 func (c *cluster) Start(ctx context.Context, t *test, opts ...option) {
        FatalIfErr(t, c.StartE(ctx, opts...))
```

Something like the above.

Reply:
I don't understand it, but I trust you more than the code that was there before :)

Done